### ISSUE TITLE
When navigating from week to week, visit both parts of a split week

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/did#765]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
When navigating from week x to week x+1, and if week x is a split week, I suggest that we navigate from week x part 1 to week x part 2, then to the next week x+1.

Currently Did is "skipping" part 2 of split weeks.

### Description
To allow for a more intuitive navigation, and avoid misunderstandings when a part of a week is skipped.

![image](https://user-images.githubusercontent.com/1837390/102504437-1f98d280-4081-11eb-86b8-eca1dee459e0.png)
Example: Add support for visiting Week 53 (January) using the arrows.

### Proposed solution

NavigateCommands: if current timesheet contains two periods do the following

if selected period is the first period

back button will set dispatch SET_SCOPE add.-w
fwd button wil set dispatch CHANGE_PERIOD payload.periods[1].id

if selected period is the second period

back button will set dispatch CHANGE_PERIOD payload.periods[0].id
fwd button will set dispatch SET_SCOPE add.+w
